packages/manager-react-components/src/components/order/Order.component.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { OrderContextProvider } from './Order.context';
+import { OrderConfiguration } from './OrderConfiguration.component';
+import { OrderSummary } from './OrderSummary.component';
+import './translations';
+
+export const Order = ({ children }: { children: JSX.Element[] }) => {
```

Comment:
It's exactly same but it's own name. I let's you juge. No impact 

```suggestion
export const Order = ({ children }: React.PropsWithChildren) => {
```